jsettlers.graphics.swing/src/jsettlers/graphics/swing/resources/ConfigurationPropertiesFile.java

```diff
@@ -38,7 +38,7 @@ public ConfigurationPropertiesFile(File file) throws FileNotFoundException,
 		this.configFile = file;
 
 		Properties defaultProperties = new Properties();
-		defaultProperties.load(ConfigurationPropertiesFile.class.getResourceAsStream("defaultConfig.prp"));
+		defaultProperties.load(ConfigurationPropertiesFile.class.getResourceAsStream("/defaultConfig.prp"));

```

Comment:
This file could be generated if it doesn't exist and then it doesn't need to be tracked.